### PR TITLE
[fix 7332] error pop-up when sending tx from wallet

### LINKED ISSUE #7332: Empty error pop-up when sending tx from Wallet to contact if public/group chat at the top on Home screen
***Type***: Bug

***Summary***: When transaction is sent from Wallet to the contact from contacts list but there is a public or group chat stands at the top of Home screen - there is an empty `Error` pop-up appears ( `Invalid push notification payload...` in logs). `/send ` message is sent in respective 1-1 chat though after transacting from Wallet.
It's all fine if the contact I send TX to is the latest chat I had with (so this contact stands at the top of Home screen). It's also fine when Dapp is the latest item on Home screen

#### Expected behavior
No errors when sending tx from Wallet to contact if public/group chat at the top on Home screen
#### Actual behavior
Empty error pop-up when sending tx from Wallet to contact if public/group chat at the top on Home screen
screenshot 2019-01-24 at 13 18 04

### Reproduction

- Open Status and recover an account with assets (Contact1)
- Create another contact (Contact2)
- Contact2 sends contact request to the Contact1 and sends text message
- Contact1 navigates to 1-1 chat and taps 'Add to contacts'
- Contact1 joins public chat (e.g. `#testpushwallet`) and sends a text message there
- Contact1 navigates to `Wallet` -> `Send`
- Contact1 selects Contact2 from `Choose recipient` -> Contact2
- Contact1 specify amount to send (e.g. 0.00001, ETH Asset stays by default)
- Contact1 signs the transaction

### Additional Information
* Status version: Develop `0.9.33 (2019012408) status-go v0.19.0-beta.1`

* Operating System: Android and iOS

#### Logs

```
01-24 11:01:24.686 27506 27616 I ReactNativeJS: ***
01-24 11:01:24.686 27506 27616 I ReactNativeJS: NAME: nil
01-24 11:01:24.686 27506 27616 I ReactNativeJS: MESSAGE: "Invalid push notification payload{:from \"0x043a5583b23ed88ef030fe4636c8cf7d5bace61f928a626726dacc9f1ece90012f5cfc7a141d883a6b86254f743fc52c8343496c28152cbe2aad920b7fbaa6e2bd\", :to nil, :id \"0x6b4e146b37eab0662ce9db71ef9864b0bf657fe3497c3197cc7158349b357394\"}"
01-24 11:01:24.686 27506 27616 I ReactNativeJS: 
01-24 11:01:24.686 27506 27616 I ReactNativeJS: 
01-24 11:01:24.686 27506 27616 I ReactNativeJS: 
01-24 11:01:24.686 27506 27616 I ReactNativeJS: ***
```

### PR DESCRIPTION
**Pull request by yenda**
when the last used chat was a public chat, the public-key of
the recipient to populate the `:to` key in the notification data
was taken from `:current-chat-id`in app-db.

this fix ensures that the right chat-id (the actual public-key of
the contact) is used instead of current-chat-id by changing the arity
of `send-push-notification`

fixes #7332 

status: ready

### CODE REVIEW

**Review by vitvly — APPROVED** [review]

**Review by mandrigin — APPROVED** [review]

**Review by cammellos — APPROVED** [review]

**Review by mandrigin — APPROVED** [review]

**Review by pedropombeiro — APPROVED** [review]

**Comment by antdanchenko** [comment]
all 7 tests are failed due to pairing popup which will be disabled for e2e builds when #7381 is merged

**Comment by asemiankevich** [comment]
do we need to wait for rerun or can merge @Serhy ? i would say yes, it works okay to me

**Comment by Serhy** [comment]
@asemiankevich  #7381 for e2e test just finished and should be merged soon. 
In order to unblock PR for merge, lets re-run failed 7 tests.
Manually it's all OK from me too!

**Comment by antdanchenko** [comment]
@yenda please rebase the PR on develop in order to include fix from #7381

**Comment by Serhy** [comment]
Let's merge!

**Pull request merged**
